cli/cmdlineparser.cpp

```diff
@@ -1605,10 +1609,22 @@ CmdLineParser::Result CmdLineParser::parseFromArgs(int argc, const char* const a
         return Result::Fail;
     }
 
+    // TODO: conflicts with analyzeAllVsConfigs
```

Comment:
We have like four different ways to select a configuration from the GUI and CLI. At least the code doing it should be unified (if possible). But something for later after the import/CLI handling has been cleaned up and has better test coverage.

Reply:
so as far as I see this will not be an "official" option to start with.. i.e. not shown in the --help output.. which will make it easier to unify later.